common/main/en.xml

```diff
@@ -9129,145 +9129,212 @@ annotations.
+			<namePattern>{surname-initial-allCaps}{given-initial-allCaps}{given2-initial-allCaps}</namePattern>
 		</personName>
 		<personName length="monogram" style="formal" order="givenFirst">
-			<namePattern>{given-initial}{given2-initial}{surname-initial}</namePattern>
+			<namePattern>{given-initial-allCaps}{given2-initial-allCaps}{surname-initial-allCaps}</namePattern>
 		</personName>
-		<personName length="monogram" style="informal" order="surnameFirst">
-			<namePattern>{surname-initial}{given-informal-initial}</namePattern>
+		<personName length="monogram" style="formal" order="surnameFirst">
+			<namePattern>{surname-initial-allCaps}{given-initial-allCaps}{given2-initial-allCaps}</namePattern>
+		</personName>
+		<personName length="monogram" style="informal" order="sorting">
+			<namePattern>{surname-initial-allCaps}{given-informal-initial-allCaps}</namePattern>
 		</personName>
 		<personName length="monogram" style="informal" order="givenFirst">
-			<namePattern>{given-informal-initial}{surname-initial}</namePattern>
+			<namePattern>{given-informal-initial-allCaps}{surname-initial-allCaps}</namePattern>
 		</personName>
-		<personName length="monogramNarrow" style="formal" order="surnameFirst">
-			<namePattern>{surname-initial}</namePattern>
+		<personName length="monogram" style="informal" order="surnameFirst">
+			<namePattern>{surname-initial-allCaps}{given-informal-initial-allCaps}</namePattern>
+		</personName>
+		<personName length="monogramNarrow" style="formal" order="sorting">
+			<namePattern>{surname-initial-allCaps}</namePattern>
 		</personName>
 		<personName length="monogramNarrow" style="formal" order="givenFirst">
-			<namePattern>{surname-initial}</namePattern>
+			<namePattern>{surname-initial-allCaps}</namePattern>
 		</personName>
-		<personName length="monogramNarrow" style="informal" order="surnameFirst">
-			<namePattern>{given-informal-initial}</namePattern>
+		<personName length="monogramNarrow" style="formal" order="surnameFirst">
+			<namePattern>{surname-initial-allCaps}</namePattern>
+		</personName>
+		<personName length="monogramNarrow" style="informal" order="sorting">
+			<namePattern>{given-informal-initial-allCaps}</namePattern>
 		</personName>
 		<personName length="monogramNarrow" style="informal" order="givenFirst">
-			<namePattern>{given-informal-initial}</namePattern>
+			<namePattern>{given-informal-initial-allCaps}</namePattern>
+		</personName>
+		<personName length="monogramNarrow" style="informal" order="surnameFirst">
+			<namePattern>{given-informal-initial-allCaps}</namePattern>
 		</personName>
-		<sampleName item="minimal">
+		<!-- The following samples are temporary, to be filled out with something better -->
+		<sampleName item="givenSurname">
+			<nameField type="prefix">∅∅∅</nameField>
 			<nameField type="given">Katherine</nameField>
+			<nameField type="given2">∅∅∅</nameField>
+			<nameField type="givenInformal">∅∅∅</nameField>
```

Comment:
this needs to be given-informal

Reply:
and elsewhere.

Reply:
I thought we said that we wanted attribute values to be camelCase (as opposed to the placeholder values, which can have appended hyper-separated modifiers)? Of course this means a mapping in the enums between the attribute value and the placeholder value).